x/denommetadata/keeper/keeper.go

```diff
@@ -0,0 +1,77 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	"github.com/dymensionxyz/dymension-rdk/x/denommetadata/types"
+)
+
+// Keeper of this module maintains distributing tokens to all stakers.
+type Keeper struct {
+	storeKey   storetypes.StoreKey
+	cdc        codec.BinaryCodec
+	paramSpace paramtypes.Subspace
+
+	bankKeeper types.BankKeeper
+	hooks      types.MultiDenomMetadataHooks
+}
+
+// NewKeeper creates new instances of the Keeper
+func NewKeeper(
+	cdc codec.BinaryCodec,
+	storeKey storetypes.StoreKey,
+	bk types.BankKeeper,
+	hooks types.MultiDenomMetadataHooks,
+	paramSpace paramtypes.Subspace,
+) Keeper {
+	// set KeyTable if it has not already been set
+	if !paramSpace.HasKeyTable() {
+		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
+	}
+
+	return Keeper{
+		storeKey:   storeKey,
+		cdc:        cdc,
+		paramSpace: paramSpace,
+		bankKeeper: bk,
+		hooks:      hooks,
+	}
+}
+
+// GetParams returns the total set of denommetadata parameters.
+func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
+	k.paramSpace.GetParamSet(ctx, &params)
+	return params
```

Comment:
```suggestion
	return
```

Reply:
in other places in the code, same `GetParams`. It does return params though, you sure we should remove it.. ?

Reply:
in Go you can just write 'return' if the return variables are already named in the signature